src/runner.js

```diff
@@ -13,19 +13,51 @@ class Runner {
     core.info(
       `Commits: Comparing the PR commits to verify that they are all from ${privileged_requester_username}`,
     );
-    for (const [, commit] of Object.entries(this.pullRequest.listCommits())) {
```

Comment:
`this.pullRequest.listCommits()` was never using `await` so I'm unsure how this was ever working in the past 😬